client/my-sites/domains/domain-management/settings/cards/domain-security-details/index.tsx

```diff
@@ -22,45 +75,105 @@ const DomainSecurityDetails = ( { domain }: DetailsCardProps ) => {
 			case sslStatuses.SSL_ACTIVE:
 				return null;
 			case sslStatuses.SSL_PENDING:
-				return translate(
-					'It may take up to a few hours to add an SSL certificate to your site. If you are not seeing it yet, give it some time to take effect.',
-					{ textOnly: true }
+				if ( sslDetails?.is_newly_registered ) {
+					return (
+						<p className="domain-security-details__description-message">
+							{ translate(
+								'It may take up to a few hours to add an SSL certificate to your site. If you are not seeing it yet, give it some time to take effect.',
```

Comment:
I think that "a few hours" is too long. Most of our domains will start resolving in less than 30 minutes.  What do you think about replacing this with something like "Your newly registered domain is almost ready! It can take up to 30 minutes for the domain to start resolving to your site so we can issue a certificate. Please check back soon."